### PR TITLE
add a contact form

### PR DESCRIPTION
**Pull request by sulaimanahmadi**
**Add following things**
1. add a contact form.
2. Collect data by using the Formspree services.
3. Implement the button interactions.

### CODE REVIEW

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @sulaimanahmadi :wave:, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

invalid

## Invalid Code Review Request

You have submitted a pull request with no commit history from your partner. This is a pair-programing activity that is intended to ensure you work together with all your partners, as you also do the same in their repositories. This prepares you to co-work effectively in a team.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -525,3 +525,137 @@ p {
...
   text-decoration: none;
 }
+
+.form-section {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  text-align: center;
+}
+
```
Comment (brainconnect93) [inline style.css:535]: - [ ] Great effort working on the `contact section` of the portfolio project, the form spree and every other things works really fine, but i notice the design and alignment of your `text` and `form` requires an attention. please kindly make sure the whole text and form are aligned centered and be sure it looks like what we have in the Figma design 👏 

| Your Design 👇  | Figma Design 👇 |
| ------------- | ------------- |
| form | temp-real |

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @sulaimanahmadi, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). ✔️
- Input fields have the `required` attribute to activate the validation ✔️
- Max-length attribute Used on the input ✔️
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @sulaimanahmadi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**